src/App.css

```diff
@@ -1,3 +1,66 @@
 @tailwind base;
 @tailwind utilities;
 @tailwind components;
+
+.sec {
+  display: flex;
+  justify-content: center;
+}
+
+.mission {
+  width: 90%;
+  height: 90%;
+}
+
+.mission,
+th,
+td {
+  height: 90%;
+  border: 1px solid black;
+  text-align: left;
+}
+
+.mission tr:nth-child(even) {
+  background-color: #f2f2f2;
+}
+
+.mission tr:hover {
+  background-color: #ddd;
+}
+
+.mis {
+  width: 10%;
+}
+
+.des {
+  width: 60%;
+}
+
+.sta {
+  width: 30%;
+}
```

Comment:
Grid system is more suitable in this case, but percentage is also fine

Reply:
thanks for your advice I am going to change right now